mypy/stubgen.py

```diff
@@ -574,6 +575,16 @@ def get_qualified_name(o: Node) -> str:
         return '<ERROR>'
 
 
+def walk_packages(packages: List[str]):
+    for package_name in packages:
+        package = __import__(package_name)
```

Comment:
I'm really not happy that this imports the target package name. At the very least you should honor the `--no-import` flag.